source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -128,17 +128,162 @@ the Server manual documentation.
+     - :manual:`$maxN </reference/operator/aggregation/maxN-array-element/>`
+
+   * - `min() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#min(T)>`__
+     - :manual:`$min </reference/operator/aggregation/min/>`
+
+   * - `minN() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#minN(com.mongodb.client.model.mql.MqlInteger)>`__
+     - :manual:`$map </reference/operator/aggregation/minN-array-element/>`
+
+   * - `multiply() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#multiply(java.util.function.Function)>`__
+     - :manual:`$multiply </reference/operator/aggregation/multiply/>`
+
+   * - `size() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#size()>`__
+     - :manual:`$size </reference/operator/aggregation/size/>`
+
+   * - `slice() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#slice(int,int)>`__
+     - :manual:`$slice </reference/operator/aggregation/slice/>`
+
+   * - `sum() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#sum(java.util.function.Function)>`__
+     - :manual:`$sum </reference/operator/aggregation/sum/>`
+
+   * - `unionArrays() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#unionArrays(java.util.function.Function)>`__
+     - :manual:`$setUnion </reference/operator/aggregation/setUnion/>`
+
+.. include:: /includes/fundamentals/agg-mql-diff.rst
+
+Suppose you have a collection of movies, each of which contains an array
+of nested documents for upcoming showtimes. Each nested document
+contains an array that represents the total number of seats in the
+theater, where the first array entry is the number of premium seats and
+the second entry is the number of regular seats. Each nested document
+also contains the number of tickets that have already been bought for
+the showtime. A document in this collection might resemble the
+following:
+
+.. code-block:: json
+   :copyable: false
+
+   {
+     "_id": ...,
+     "movie": "Hamlet",
+     "showtimes": [
+       {
+         "date": "May 14, 2023, 12:00 PM",
+         "seats": [ 20, 80 ],
+         "ticketsBought": 100
+       },
+       {
+         "date": "May 20, 2023, 08:00 PM",
+         "seats": [ 10, 40 ],
+         "ticketsBought": 34
+       }
+     ]
+   }
+
+You want to see only showtimes that still have available tickets.
+You can calculate the total number of seats by taking the sum of the
+``seats`` array by using the ``sum()`` method. To
+filter for showtimes with available tickets, pass a predicate to the
+``filter()`` method that checks if ``ticketsBought`` is less
+than the total number of seats by using the ``lt()``
+method. To output the showtimes with available tickets as the
+``availableShowtimes`` array, use the ``project()`` method.
+
```

Comment:
```suggestion
To see only showtimes with available tickets, pass a predicate to the
``filter()`` method.
Within this predicate, calculate the total number of seats by taking the sum of the
``seats`` array by using the ``sum()`` method. 
Check if ``ticketsBought`` is less
than the total number of seats by using the ``lt()``
method. To output the showtimes with available tickets as the
``availableShowtimes`` array, use the ``project()`` method.
```

Rearranged because the total calculation is performed inside ("after") the predicate. The rearrangement made the first line redundant.

Might add a note like: "To increase legibility, this example assigns intermediary results to the totalSeats and isAvailable variables. Inlining the variables produces equivalent results." since readers might be surprised to see variables, and wonder if they really work as intended.